config/config_test.go

```diff
@@ -33,6 +33,8 @@ func TestConfig(t *testing.T) {
 					ZebedeeURL:                 "http://localhost:8082",
 					EnablePermissionsAuth:      true,
 					DatasetAPIURL:              "http://localhost:22000",
+					OTExporterOTLPEndpoint:     "localhost:4317",
+					OTServiceName:              "dp-population-types-api",
```

Comment:
needs batch timeout - see my message in the core team channel

Reply:
Added batch timeout, updated test, updated readme